### PR TITLE
Add cool css animation

### PR DESCRIPTION
**Pull request by DenseOriginal**
Added this cool moving gradiant animation

### CODE REVIEW

File: animations/text-color-gradiant.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  background-size: auto auto;
+  background-clip: border-box;
+  background-size: 200% auto;
+  color: #fff;
+  background-clip: text;
+  text-fill-color: transparent;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  animation: textclip 2s linear infinite;
+  display: inline-block;
```
Comment (NiallEccles) [inline animations/text-color-gradiant.css:19]: Can you remove the `display: inline-block;` ?

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Hi 👋 Thanks for making this PR.

Can you update this PR to address the issue I have highlighted?

File: animations/text-color-gradiant.css (hunk 1 above)
Comment (DenseOriginal) [reply animations/text-color-gradiant.css]: Done

**Comment by NiallEccles** [comment]
Nice! I love the colours 👏

**Pull request merged**